web3.md

```diff
@@ -186,5 +193,37 @@ module WEB3
            <code> CODE </code>
            ...
          </account>
+
+    syntax KItem ::= "#evm_snapshot"
+ // --------------------------------
+    rule <k> #evm_snapshot => #pushNetworkState ~> #sendResponse( "result" : #unparseQuantity( size ( SNAPSHOTS ) ) ) ... </k>
+         <snapshots> SNAPSHOTS </snapshots>
```

Comment:
Is this supposed to send the number of snapshots before or after the `#pushNetworkState` operation?

Reply:
It should return the `id` of the snapshot. So by returning the size of the snapshot list after `#pushNetworkState` I tried to send back the position of the newly added snapshot.